DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/projectingUsingFunctions/projectingUsingFunctions.java

```diff
@@ -0,0 +1,48 @@
+//region Usings
+import net.ravendb.client.documents.session.IDocumentSession;
+import net.ravendb.client.documents.session.IRawDocumentQuery;
+//endregion
+import net.ravendb.demo.common.DocumentStoreHolder;
+
+
+
+import java.util.List;
+
+public class projectingUsingFunctions {
+
+    //region Demo
+    public static class EmployeeDetails {
+        public String FullName;
+        public String Title;
+    }
+    //endregion
+
+
+    public List<EmployeeDetails> run() {
+        List<EmployeeDetails> projectedResults;
+        //region Demo
+        try (IDocumentSession session = DocumentStoreHolder.store.openSession()) {
+        //region Step_1
+            String rawQueryString =  "declare function output(employee) {" +
```

Comment:
Remove the extra space after =

Reply:
format like in the GO demo, without `"` and make similar indentations

![image](https://user-images.githubusercontent.com/17376974/148752816-0cbbc8d2-e4ab-4029-93ec-b26fa6ced027.png)

Reply:
Done